app/tasks/templates/compliance-check-specialist.html

```diff
@@ -182,6 +183,25 @@ <h2 class="document-title">Compliance check</h2>
                         {% if brief.data.securityClearance == 'mustHave'%}
                             <td>{{ response.data.get('securityClearance', 'Unknown') }}</td>
                         {% endif %}
+                        {% if response.supplier.data.get('recruiter', '') == 'no'%}
+                            <td>
+                                N/A
```

Comment:
I wonder if this should be the same message as below when no labour hire licence is detected. We do allow a consultancy only supplier to respond to a specialist brief, and just because they are a consultancy shouldn't mean labour hire licence details are not applicable - if they are responding to a specialist brief then it is applicable (in which case, they really should update their profile to indicate they are a supplier of labour hire, but they don't actually have to).

Reply:
Not sure that I agree with this one, since the seller wasn't given the opportunity to provide that information